tests/installation/ipxe_install.pm

```diff
@@ -148,13 +150,13 @@ sub run {
         sleep 120;
     } else {
         select_console 'sol', await_console => 0;
-        sleep 300;
-        my $ssh_vnc_wait_time = 1200;
+        my $ssh_vnc_wait_time = 1500;
         my $ssh_vnc_tag       = eval { check_var('VIDEOMODE', 'text') ? 'sshd' : 'vnc' } . '-server-started';
         my @tags              = ($ssh_vnc_tag);
         if (check_screen(\@tags, $ssh_vnc_wait_time)) {
             save_screenshot;
             sleep 2;
+            prepare_disks if (!is_upgrade && !get_var('KEEP_DISKS'));
```

Comment:
`prepare_disks` does calls [`wipefs`](https://man7.org/linux/man-pages/man8/wipefs.8.html), which is probably fast.
But just a curious question: why this? (haven't found the reason in commit message, maybe it's obvious and I don't see it.

Reply:
We want clean machine before installation, to avoid not necessary situations (questions about upgrade, filesystems with encryption) in installer, which detects previously installed system. `prepare_disks` is already used in existing other `pxe` and `spvm` scenarios.

Reply:
Understand, np :). My question was practical, that when ipxe installation was broken it didn't destroy the machine. One then could just reboot machine and do manual setup (I did it several weeks). But suppose ipxe installation will be now working, so this is irrelevant :)
(and I noticed the variable to avoid that).